charts/steward/templates/clusterrole-tenant-controller.yaml

```diff
@@ -12,14 +12,14 @@ rules:
   resources: ["clusterroles"]
   verbs: ["bind"]
   resourceNames:
-  {{- toYaml .Values.tenantController.possibleTenantRoles | nindent 4 }}
+  {{- toYaml .Values.tenantController.possibleTenantRoles | nindent 2 }}
```

Comment:
@anbrsap Is this by intention? Shouldn't it be 4 spaces indented?

Reply:
It's a Yaml list, so the indentation of 2 works here. But 4 is even better, as it would prevent the injection of further fields on the same level as `resourceNames` by specifying `.Values.tenantController.possibleTenantRoles` as a map.

I'll change that.